Sources/BraintreeShopperInsights/BTShopperInsightsClient.swift

```diff
@@ -69,16 +76,23 @@ public class BTShopperInsightsClient {
                 isVenmoRecommended: venmo?.recommended ?? false,
                 isEligibleInPayPalNetwork: payPal?.eligibleInPayPalNetwork ?? false || venmo?.eligibleInPayPalNetwork ?? false
             )
-            return self.notifySuccess(with: result)
+            return self.notifySuccess(with: result, for: experiment)
         } catch {
-            throw self.notifyFailure(with: error)
+            throw self.notifyFailure(with: error, for: experiment)
         }
     }
 
     /// Call this method when the PayPal button has been successfully displayed to the buyer.
     /// This method sends analytics to help improve the Shopper Insights feature experience.
-    public func sendPayPalPresentedEvent() {
-        apiClient.sendAnalyticsEvent(BTShopperInsightsAnalytics.payPalPresented)
+    /// - Parameters:
+    ///    - paymentMethodsDisplayed: Optional:  The list of available payment methods, rendered in the same order in which they are displayed i.e. ['Apple Pay', 'PayPal']
+    ///    - experiment: Optional:  A `JSONObject` passed in as a string containing details of the merchant experiment.
+    public func sendPayPalPresentedEvent(paymentMethodsDisplayed: String? = nil, experiment: String? = nil) {
```

Comment:
Do you think instead of asking merchants to turn their array into a string we can take an array of strings and join it into a string behind the scenes?

Reply:
yep! addressed in 9af9d4ac860ff0497a192b3d02150ca16877e41f